install/docker/debian-docker-install.md

```diff
@@ -0,0 +1,97 @@
+# Install Docker for Kata Containers on Debian
+
+> **Note:**
+>
+> - This guide assumes you have
+>   [already installed the Kata Containers packages](../debian-installation-guide.md).
+> - this guide allows for installation with `systemd` or `sysVinit`init systems 
+> - notes are also provided for running multiple runtimes
+> - tested with `docker-ce=17.12.0~ce-0~debian`
```

Comment:
Nit: I'd remove this bullet and include the details in the commit message if you wish.

Reply:
removed